src/middleware/validation.js

```diff
@@ -29,17 +31,45 @@ class validation {
  * @param  {next} next
  * @returns {next} next
  */
-  static question(req, res, next) {
+  static getQuestion(req, res, next) {
```

Comment:
Nice one, I like the fact that you modified the method name here and the rest above to be more descriptive.